optimum/intel/openvino/modeling_decoder.py

```diff
@@ -575,11 +576,18 @@ def _from_pretrained(
             local_files_only=local_files_only,
         )
 
+        # Give default quantization config if not provided and load_in_8bit=True
+        if load_in_8bit:
+            quantization_config = quantization_config or {"bits": 8}
+
         if isinstance(quantization_config, dict):
+            if quantization_config == {"bits": 4} and config.name_or_path in _DEFAULT_4BIT_CONFIGS:
```

Comment:
thanks, @echarlaix. I'm fine with merging this if it works.

Reply:
perfect, will add the config saving + test mentioned above in a following PR